src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final int DEF_POS = 0;
+
+    private int size;
+    private Object[] elements;
+
+    public ArrayList() {
+        elements = new Object[DEFAULT_SIZE];
+        size = 0;
+    }
+
+    private Object[] grow() {
```

Comment:
```suggestion
    private void grow() {
```
let this method change the array
also lets move grow to the end - private methods should be after public methods